### DISCUSSION TITLE
User Agreement and Privacy Agreement

### DISCUSSION THREAD

**Discussion by liuyuzhou-it** [zixpo/candybar#141] [Ideas]
It is hoped to provide a user agreement and privacy agreement when the application is launched for the first time, informing users of the purposes for which the application needs to obtain permission, and providing options for agreeing and disagreeing. If you agree, you will enter the application normally, and if you disagree, you will exit the application.

**Comment by Donnnno** [comment]
Right now it's explained in the faq, if you scroll down. It's good to know tho, that by default no data is collected
